lib/oban/migrations.ex

```diff
@@ -333,4 +333,30 @@ defmodule Oban.Migrations do
       record_version(prefix, 5)
     end
   end
+
+  defmodule V7 do
+    @moduledoc false
+
+    use Ecto.Migration
+
+    import Oban.Migrations.Helper
+
+    def up(prefix) do
+      create_if_not_exists index(
+                             :oban_jobs,
+                             ["attempted_at desc", :id],
```

Comment:
Why do you need to index the `:id` column?

Reply:
Including `id` allows an index only scan because the only field we are selecting is the `id`.

Reply:
Thank you. TIL.